frontend/src/app/users/users-profile/users-profile.component.ts

```diff
@@ -0,0 +1,68 @@
+/*
+ * SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { BehaviorSubject } from 'rxjs';
+import { BreadcrumbsService } from 'src/app/general/breadcrumbs/breadcrumbs.service';
+import { Project } from 'src/app/projects/service/project.service';
+import { User, UserService } from 'src/app/services/user/user.service';
+
+@Component({
+  selector: 'app-users-profile',
+  templateUrl: './users-profile.component.html',
+  styleUrls: ['./users-profile.component.css'],
+})
+export class UsersProfileComponent implements OnInit, OnDestroy {
+  user: User | undefined;
+  userId: number | undefined;
+  commonProjects = new BehaviorSubject<Project[] | undefined>(undefined);
+  joinedString: string | undefined;
+
+  public readonly commonProjects$ = this.commonProjects.asObservable();
+
+  constructor(
+    public userService: UserService,
+    private route: ActivatedRoute,
+    private breadcrumbsService: BreadcrumbsService,
+  ) {}
+
+  ngOnInit() {
+    this.userService.updateOwnUser();
+    this.route.queryParams.subscribe((params) => {
+      this.userId = +params.userId;
+      if (this.userId) {
+        this.userService.getUserById(this.userId).subscribe((user) => {
+          this.user = user;
+          this.breadcrumbsService.updatePlaceholder({ user: user });
+          if (this.userId !== this.userService.user?.id) {
+            this.userService.loadCommonProjects(this.userId!).subscribe({
+              next: (projects) => this.commonProjects.next(projects),
+              error: () => this.commonProjects.next(undefined),
+            });
+          } else {
+            this.userService
+              .getUserHistory(this.userService.user!)
+              .subscribe((history) => {
+                if (history.created) {
+                  this.joinedString = `You joined the collaboration manager in ${new Date(
+                    history.created,
+                  )
```

Comment:
![image](https://github.com/DSD-DBS/capella-collab-manager/assets/23395732/31ee314c-8446-49ab-a733-a9e6d81d9d0a)

I'd move this up. The text should be below the username.